### ISSUE TITLE
HBO wrongly listed as DK provider of Kill Bill

### ISSUE THREAD

**Issue by wonderfulspam** [streamchaser/streamchaser#286]
### Before reporting

- [X] I have tested with the lastest _master_

### Describe the bug

![Screenshot_2022-09-22-19-24-38-774_org mozilla firefox](https://user-images.githubusercontent.com/3662366/191812820-e1d55c20-617e-4982-9386-ee9b0b398985.jpg)

![Screenshot_2022-09-22-19-28-05-420_com hbo hbonow](https://user-images.githubusercontent.com/3662366/191812890-84ae36e9-9a5a-427a-8756-dfdc2d00a6ca.jpg)

### To Reproduce

Search for Kill Bill on Streamchaser and HBO.

### Expected behavior

HBO is not listed as a provider.

**Comment by Primdahl26** [comment]
Hbo and Hbo max was merged if i recall correctly. Think we need to remind TMDb if this since they also list the same providers. Justwatch seems to have the correct data though, so TMDb should have updated? @AndreasPB

**Comment by AndreasPB** [comment]
I think they used to have the same name and logo, so it was just a duplicate, and I think we handled that by only keeping uniques. But we should probably make sure that fix is doing what we think it should :D

**Comment by AndreasPB** [comment]
TMDB has the same problem on its own page https://www.themoviedb.org/movie/24-kill-bill-vol-1/watch 

I've reported it to them, but since TMDB gets their provider data from JustWatch they might not even be able to fix it... 🤦

**Comment by AndreasPB** [comment]
There we go 
![image](https://user-images.githubusercontent.com/43907402/193410948-54c45f31-bd57-484e-bda7-351bc7ba39a5.png)

**Comment by AndreasPB** [comment]
I'll check if it's still an issue tomorrow - If it is I'll try to contact JustWatch

**Comment by Primdahl26** [comment]
Justwatch displays correct data though 

https://www.justwatch.com/dk/movie/kill-bill-vol-1

**Comment by AndreasPB** [comment]
Yep, look at the discussion I had with the mod, I linked it under the picture

**Comment by AndreasPB** [comment]
It has been "fixed" on our end, but the problem is still there... We are just hiding it :D 

I'll let the issue stay open and keep trying to get TMDB to fix it
